kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/marketplace/view/MarketplaceActivity.java

```diff
@@ -141,8 +141,13 @@ public void navigateToOrderHistory() {
     }
 
     @Override
-    public void showOfferActivity(String content, String offerID) {
-        navigateToActivity(PollWebViewActivity.createIntent(this, content, offerID));
+    public void showOfferActivity(String content, String offerID, String title) {
+        PollBundle pollBundle = new PollBundle().setJsonData(content).setOfferID(offerID).setTitle(title);
```

Comment:
this code need to be in the presenter where it can be test - showOfferActivity need to get PollBundle as parameter PollWebViewActivity need to provide PollWebViewActivity.createIntent(this, PollBundle pollBundle)